### ISSUE TITLE
Crash in multilingual website after saving

### ISSUE THREAD

**Issue by tiberiuichim** [plone/volto#3649]
On a multilingual site, when saving, I get a crash

```
Cannot read properties of undefined (reading 'translations')

    at Toolbar (http://localhost:3001/static/js/client.chunk.js:70824:5)
```

Logging the `this.props.content`, I can see that at one point the content becomes:

```
lock: {locked: false, stealable: true}
```

**Comment by pnicolli** [comment]
Yeah I discussed it at the Volto Team Meeting two weeks ago. At first I thought it was about overriding the content, but it's actually because the content is reset on route change, then the unlock_content reply comes in and you get the situation when you only have the lock info in the content, then the get_content info comes in from the backend and overrides the lock info. If the page is rendered before you receive the get_content reply, the error is thrown.

Fixed in #3685

**Comment by davisagli** [comment]
This was fixed a couple years ago in https://github.com/plone/volto/pull/4377
